src/tester/test.ts

```diff
@@ -22,6 +22,15 @@ async function main(options: TesterOptions, nProcesses: number, all: boolean): P
     const log = loggerWithErrors()[0];
     const dt = await getDefinitelyTyped(options, log);
     await parseDefinitions(dt, { nProcesses, definitelyTypedPath: options.definitelyTypedPath }, log);
-    await checkParseResults(/*includeNpmChecks*/false, dt, options, new UncachedNpmInfoClient());
+    try {
+        await checkParseResults(/*includeNpmChecks*/false, dt, options, new UncachedNpmInfoClient());
+    }
+    catch (e) {
+        if (!all) {
+            (await getAffectedPackagesFromDiff(dt, options.definitelyTypedPath, "affected")).dependentPackages.map(t => t.desc);

```

Comment:
Did you mean to do something with the result of this expression, or do you not need the `.dependentPackages.map(...)`, or am I missing something?

Reply:
I meant to use it until I noticed that getAffectedPackagesFromDiff already logged the information. Removed!